### PR TITLE
Add Phoenix Prize

### PR DESCRIPTION
**Pull request by phoe**
I'd like to throw a few cents and to incentivize reviving OLM a little bit. What do you think?

Feel free to edit the wording to your liking, the general idea should be there.

### CODE REVIEW

File: content/post/20211129-aoc-contest.md [hunk 1]
```diff
@@ -132,4 +134,9 @@ If it's truly the case that a problem is unrealistic to solve with Coalton alone
 
 ## We're Here to Help
 
-We know Coalton still needs work. That's why we are here to help, suggest directions, give tips, and answer questions. Please visit us on [Discord](https://discord.gg/cPb6Bc4xAH) to chat about Coalton!
\ No newline at end of file
+We know Coalton still needs work. That's why we are here to help, suggest directions, give tips, and answer questions. Please visit us on [Discord](https://discord.gg/cPb6Bc4xAH) to chat about Coalton!
+
+
+## Fine Print
```
Comment (stylewarning) [inline content/post/20211129-aoc-contest.md:140]: Maybe just add this to the above fine print. :)

**Review by stylewarning — CHANGES_REQUESTED** [review]
tiny change request

**Comment by phoe** [comment]
Done; should be possible to squash&rebase it.

**Review by stylewarning — APPROVED** [review]
Exciting!

**Pull request merged**